src/chapter4/challenges.md

```diff
@@ -1 +1,63 @@
 # Challenges
+
+## Task 1 - Parallise for Loop
+
+Goal: To to create an array [0,1,2………...19]
+
+1. Git clone https://github.com/Yusuke710/HPC_training2021.git 
+2. Go to the directory “question”. Compile array.c and execute it. Check the run time of the serial code
+3. Add `#pragma<>` 
+4. Compile the code again
+5. Run parallel code and check the improved run time
+
+## Task 2 - Run task 1 on HPC cluster
+
+1.  Check the available partitions with `show_cluster`
+2.  Modify `RunHello.sh `
+3.  `sbatch RunHello.sh`
+4.  `cat slurm<>.out` and check the run time
+
+>[!note]
```

Comment:
Make sure the Markdown is correct otherwise it won't render correctly. `>` means quote block and `[]()` is a link.